pkg/synopsysctl/cmd_deploy.go

```diff
@@ -314,6 +331,15 @@ var deployCmd = &cobra.Command{
 			return err
 		}
 
+		// Create default sizes
+		defaultsSizes := []string{"small", "medium", "large"}
+		for _, v := range defaultsSizes {
+			_, err = sizeClient.SynopsysV1().Sizes(operatorNamespace).Create(size.GetSize(v))
+			if err != nil {
```

Comment:
Should probably just warn rather than error here.  If a previous operator was running then size crs could still exist.  destroy doesn't seem to be cleaning up the size CRs, so it seems like this is a feasible case.

A question to answer is should the destroy delete the size crs?

Reply:
I think, destroy should remove the crd and cr's

Reply:
destroy, adding create/update/delete size should be separate PR imo...

Reply:
@rrati approve pls

Reply:
It already does remove the size CRD

Reply:
destroy should delete the crds and crs in this impl, and that seems a sane path.